3.10/data-science.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: default
+description: Scalable Graph Analytics and GraphML
+---
+Data Science
+==================
+
```

Comment:
A [lead paragraph](https://github.com/arangodb/docs#adding-a-lead-paragraph) would be nice, maybe using the first sentence of the content.
See e.g. https://github.com/arangodb/docs/blob/main/3.10/use-cases.md?plain=1#L3-L12 (ignore the `page-toc` part).